Tasks/NuGetCommand/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
```

Comment:
Nit. changing the default NuGet version probably warrants a minor version update.